core/src/main/resources/schema/mysql/mysql-event-tag-migration.sql

```diff
@@ -0,0 +1,34 @@
+-- add new column
+ALTER TABLE event_tag
+    ADD persistence_id  VARCHAR(255),
+    ADD sequence_number BIGINT;
+-- migrate rows
```

Comment:
I just realised that the script needs to be run in two parts. First add the columns, redeploy with new version, then run the rest of the script and redeploy once more with legacy-mode set to false. See my comment on the migration guide PR: https://github.com/akka/akka-persistence-jdbc/pull/781#pullrequestreview-1698857285